tests/test_basic/results.py

```diff
@@ -3,14 +3,13 @@
 import sys
 
 # import statepoint
-sys.path.insert(0, '../../src/utils')
```

Comment:
I think the reason the Travis CI builds are failing is because of this. `from openmc.statepoint import StatePoint` probably fails because it doesn't find openmc anywhere on the python path.